controllers/lmes/lmevaljob_controller.go

```diff
@@ -168,7 +184,7 @@ func (r *LMEvalJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		job.Status.State = lmesv1alpha1.NewJobState
 	}
 
-	if job.Spec.Suspend {
+	if job.Spec.Suspend && JobMgrEnabled {
```

Comment:
What should we do if `JobMgrEnabled` is `true` but there is no `kueue.x-k8s.io/queue-name` label in the LMEvalJob? Does 'no Kueue label' mean using a regular approach to create a pod without Kueue's resource management?

Reply:
Done. Per discussion no label should mark job complete with failed reason and messages.